be/src/pipeline/exec/spill_sort_sink_operator.cpp

```diff
@@ -179,9 +179,20 @@ Status SpillSortSinkOperatorX::sink(doris::RuntimeState* state, vectorized::Bloc
             local_state._shared_state->in_mem_shared_state->sorter->data_size());
     if (eos) {
         if (_enable_spill) {
```

Comment:
I think   spill_sort_sink_operator is used when _enable_spill== true. So that I think this variable is always true in this case??

Reply:
We currently only support spill for full sort, but when creating the sort operatorX, it's hard to decide whether it's full sort.

It's in the `SortSinkOperatorX::prepare` phase to decide whether to use full sort, so we create 
 `SpillSortSinkOperatorX` when `enable_sort_spill=true` for all type of sort, but actually only enable spillable sort for full sort.